burn-book/src/building-blocks/dataset.md

```diff
@@ -0,0 +1,81 @@
+# Dataset
+
+Most deep learning training is done on a dataset, with perhaps the exception of reinforcement
```

Comment:
I would replace the two commas with a pair of _tirets demi-cadratin_, as we say in french: 
[...] is done on a dataset –with perhaps the exception of reinforcement learning– so it's essential [...]

Reply:
In fact I'd rewrite the sentence as:
Most deep learning training being done on datasets –with perhaps the exception of reinforcement learning–, it is essential to provide a convenient and performant API.